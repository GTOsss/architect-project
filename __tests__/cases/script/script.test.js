const appRoot = process.cwd();
const { resolve } = require('path');
const {
  returnOutputPathsWithContent,
  promisifyCliCommand,
  SnapshotSerializer,
} = require('../../utils/utilsForArcStart');
const { pathForCommand } = require('../casesConfigPath');

const currentCommand = pathForCommand.script;

describe('script', () => {
  test('test call function from script', async () => {
    await promisifyCliCommand(`arc s -c ${currentCommand}`);
    const currentPathToOutput = resolve(appRoot, currentCommand, 'output');

    SnapshotSerializer();

    expect(returnOutputPathsWithContent(currentPathToOutput)).toMatchSnapshot();
  });
});
