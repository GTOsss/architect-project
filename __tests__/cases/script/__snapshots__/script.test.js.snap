// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`script test call function from script 1`] = `
Object {
  "src": Array [
    Object {
      "content": " const options = {
    escapeString: false,
    indent: 2,
    escapeRegex: false,
  };


",
      "path": "*/script/output/src/services/name/index.ts",
    },
  ],
}
`;
